### ISSUE TITLE
see image

### ISSUE THREAD

**Issue by jonasrhee1208wastaken** [cloudlink-omega/extension#2]
## Expected Behavior

Connected to signaling server should be "true"
## Current Behavior

It is instead false
## Steps to Reproduce

Well, look at the image

I have read the directions.
![Screenshot 2024-05-07 9 45 05 AM](https://github.com/cloudlink-omega/extension/assets/121138564/75282d17-855a-4bce-a739-acc877b88201)
![Screenshot 2024-05-07 9 44 56 AM](https://github.com/cloudlink-omega/extension/assets/121138564/75a7f775-3693-4f14-acf4-1a8ad8cdd364)

**Comment by jonasrhee1208wastaken** [comment]
is there penguinmod support?

**Comment by MikeDev101** [comment]
Will investigate this issue. I've noticed penguinmod having issues with my extensions, might be related.

**Comment by jonasrhee1208wastaken** [comment]
ill tell you the user agent also

**Comment by jonasrhee1208wastaken** [comment]
{ "platform": "Chrome OS", "platformVersion": "15786.60.0" }

**Comment by jonasrhee1208wastaken** [comment]
as proof im not slandering you, i put the before and after, and the executed variable

**Comment by jonasrhee1208wastaken** [comment]
oh yeah theres a problem with registering too with your bot and your block

**Comment by MikeDev101** [comment]
Sorry for the delay, this was likely an intermittent server issue. 
![image](https://github.com/user-attachments/assets/b2fb0d91-5332-4548-a88f-47ced66e1ce7)

I recently implemented a security requirement that all accounts must have valid email addresses and verification is mandatory. Anyone who wasn't an active developer was removed, so you will most likely need to re-register an account (Check your spam inbox for the verification link).

Reopen if this issue still persists.
